apps/evm/src/clients/api/queries/getPoolDelegateApprovalStatus/index.ts

```diff
@@ -0,0 +1,19 @@
+import { IsolatedPoolComptroller } from 'libs/contracts';
+
+export interface GetNativeTokenGatewayDelegateApprovalInput {
+  poolComptrollerContract: IsolatedPoolComptroller;
+  delegateAddress: string;
+  accountAddress: string;
+}
+
+const getPoolDelegateApprovalStatus = async ({
+  poolComptrollerContract,
+  delegateAddress,
+  accountAddress,
+}: GetNativeTokenGatewayDelegateApprovalInput) => {
+  const res = await poolComptrollerContract.approvedDelegates(accountAddress, delegateAddress);
+
+  return res;
```

Comment:
To follow the same convention as other hooks, I'd recommend returning an object here (it makes it easier to extend the return type without having to refactor every piece of code that uses this hook):
```suggestion
  const isDelegateeApproved = await poolComptrollerContract.approvedDelegates(accountAddress, delegateAddress);

  return {
    isDelegateeApproved,
  };
```